doris/benchmark.sh

```diff
@@ -6,10 +6,11 @@ set -ex
 # Install
 ROOT=$(pwd)
 
+
 if [[ -n "$1" ]]; then
     url="$1"
 else
-    url='https://doris-release.s3.us-east-1.amazonaws.com/1.2/doris-1.2.0.alpha-x86_64.tar.gz'
+    url='https://doris-clickbench.oss-ap-southeast-1.aliyuncs.com/doris-linux_x64.tar.gz'
```

Comment:
Why this link is not just 
`https://apache-doris-releases.oss-accelerate.aliyuncs.com/apache-doris-2.0.3-bin-x64.tar.gz`?